pkg/s3select/select-datatypes.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Minio Cloud Storage, (C) 2015, 2018 Minio, Inc.
```

Comment:
Remove 2015, this is a new file only needs 2018

Reply:
Modified